docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -188,10 +235,9 @@ $ xtrabackup --prepare --target-dir=/data/backup \
 --keyring-file-data=/var/lib/mysql-keyring/keyring
```

Comment:
@alina-derkach-oaza sorry for the late comment. keyring-file-data is wrong for prepare component version. It should be --component-keyring-config

Reply:
below text mentions this but example remained outdated.